GCRCatalogs/dc2_coadd.py

```diff
@@ -12,16 +12,66 @@
 
 __all__ = ['DC2CoaddCatalog']
 
+FILE_PATTERN = r'merged_tract_\d+\.hdf5'
+GROUP_PATTERN = r'coadd_\d+_\d\d$'
+
+
+def calc_cov(ixx_err, iyy_err, ixy_err):
+    """Calculate the covariance between three arrays of second moments
+
+    Args:
+        ixx_err (float): The error in the second moment Ixx
+        iyy_err (float): The error in the second moment Iyy
+        ixy_err (float): The error in the second moment Ixy
+
+    Returns:
+        The covariance between the given arrays
+    """
+    # If need be, this can be generalized using a mesh grid
+    out_data = np.array([
```

Comment:
The DPDD intends `ICov` as a six-element array containing the six unique elements of the cross-band or reference band covariance matrix. In the coadd data I'm taking this to refer to whatever band was used in determining the shape slot values for a given object.

Reply:
I believe @EiffL was asking about the order of the array.  Is it
1. `[xx_xx, xy_xy, yy_yy, xx_xy, xy_yy, xx_yy]`
or
2. `[xx_xx, xx_xy, xx_yy, xy_xy, xy_yy, yy_yy]`

As @EiffL mentions, this order is not defined in the DPDD.  It will be defined in any particular schema, and we should do it in our own.  I would argue for 1.

Reply:
@EiffL and @wmwv I've just changed ICov it to adopt the second order `[xx_xx, xx_xy, xx_yy, xy_xy, xy_yy, yy_yy]`

Reply:
Could you add the documentation of the order it returns to the function docstring?

Reply:
great! thanks :-) Yes, my question was more about the order, this looks good